src/app/(vns)/page.tsx

```diff
@@ -38,7 +35,7 @@ export default function MainPage() {
                 </div>
                 <Image
                     alt={"header background"}
-                    className={`relative m-0 p-0 ${theme === "light" ? "" : "brightness-60"} z-0 w-full`}
```

Comment:
next time don't do this, Tailwind need all classnames to be available on-prem in order to filter.

if you want to use conditional CSS, `clsx` exists solely for that reason.

also we have `dark:` FYI.